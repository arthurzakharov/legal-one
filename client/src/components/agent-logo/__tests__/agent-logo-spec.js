import React from 'react';
import renderer from 'react-test-renderer';
import AgentLogo from '../agent-logo.component';

describe('AgentLogo: ', () => {
  test('render with no props passed', () => {
    const component = renderer.create(<AgentLogo />).toJSON();
    expect(component).toMatchSnapshot();
  });
  test('render with all props passed', () => {
    const component = renderer.create(<AgentLogo name="x" src="x" />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
