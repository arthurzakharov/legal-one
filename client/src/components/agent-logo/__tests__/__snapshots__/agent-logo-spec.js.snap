// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AgentLogo:  render with all props passed 1`] = `
<img
  alt="Logo-x"
  className="AgentLogo"
  src="x"
/>
`;

exports[`AgentLogo:  render with no props passed 1`] = `
<img
  alt="Logo-undefined"
  className="AgentLogo"
/>
`;
